src/cli/env.js

```diff
@@ -1,3 +1,9 @@
 export const parseEnv = () => {
-    // Write your code here 
-};
\ No newline at end of file
+  Object.entries(process.env).forEach(([prop, value]) => {
+    if (prop.match(new RegExp('RSS_'))) {
```

Comment:
```suggestion
    if (prop.match(new RegExp('^RSS_'))) {
```
Если использовать регулярки, то тут нужно добавить спецсимвол начала строки, чтобы работало правильно. Иначе посчитаются валидными и строки, где 'RSS' где-нибудь посередине, а нужно было, чтобы имя начиналось с 'RSS'